src/functions-reference/positive_continuous_distributions.Rmd

```diff
@@ -368,11 +368,13 @@ For a description of argument and return types, see section
 
 ### Probability density function
 
-If $\alpha \in \mathbb{R}^+$ and $\beta \in \mathbb{R}^+$, then for $y
+If the shape parameter $\alpha \in \mathbb{R}^+$ and the rate (or inverse scale) parameter $\beta \in \mathbb{R}^+$, then for $y
 \in \mathbb{R}^+$, \[ \text{Gamma}(y|\alpha,\beta) =
 \frac{\beta^{\alpha}}      {\Gamma(\alpha)} \, y^{\alpha - 1}
 \exp(-\beta \, y) . \]
 
+Under the shape and rate formulation of the Gamma distribution, $E(y) = \alpha/\beta$ and $\var(y) = \alpha/\beta^2$.
```

Comment:
Sorry I didn't see this before.  This is fine, but we've used `\mathbb{E}[y]` for expectations and typically `Y` for random variables.  Also `\mathrm{var}[y]` (though that may be what `\var` does?).